Makefile

```diff
@@ -5,6 +5,8 @@
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
 VERSION ?= 0.1.0
 
+CONTAINER_CMD ?= docker
```

Comment:
you missed to change https://github.com/ansible/ansible-ai-connect-operator/blob/main/Makefile#L229 (--container-tool)